File: app/src/main/java/co/early/password123/ui/about/CardLineText.java
package co.early.password123.ui.about;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.LinearLayout;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import co.early.password123.R;


public class CardLineText extends LinearLayout {

    @BindView(R.id.aboutcardline_texticon_textview)
    protected TextView textIconView;

    @BindView(R.id.aboutcardline_text_textview)
    protected TextView textView;



    public CardLineText(Context context) {
        super(context);
    }

    public CardLineText(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public CardLineText(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }



    @Override
    protected void onFinishInflate() {
        super.onFinishInflate();

        ButterKnife.bind(this, this);

    }


    public TextView getTextIconView(){
        return textIconView;
    }

    public TextView getTextView(){
        return textView;
    }

}

File: app/src/main/java/co/early/password123/ui/common/widgets/CardLineText.kt
package co.early.password123.ui.common.widgets

import android.content.Context
import android.util.AttributeSet
import android.widget.LinearLayout
import android.widget.TextView
import kotlinx.android.synthetic.main.common_cardline_text.view.*


class CardLineText : LinearLayout {


    var textIconView: TextView? = null
    var textView: TextView? = null


    constructor(context: Context) : super(context) {}

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {}

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {}


    override fun onFinishInflate() {
        super.onFinishInflate()

        textView = aboutcardline_text_textview
        textIconView = aboutcardline_texticon_textview

    }

}
